portal-api/src/v1/users/validation/rules/select-at-least-one-bank.test.js

```diff
@@ -0,0 +1,36 @@
+const selectAtLeastOneBank = require('./select-at-least-one-bank');
+
+describe('selectAtLeastOneBank', () => {
+  const selectAtLeastOneBankError = [
+    {
+      bank: {
+        order: '4',
+        text: 'Bank is required',
+      },
+    },
+  ];
+
+  const allBankError = [
+    {
+      bank: {
+        order: '4',
+        text: 'Only admins can have "All" as the bank',
+      },
+    },
+  ];
+
+  const testCases = [
```

Comment:
Same as above.

Reply:
In test cases for `select-at-least-one-bank` which has `user`, `change`, `expected`. For `inputs` just 1 condition is given so not sure if inputs can be used if not there will  3 different tests without for each loop.